test/apicoverage-webhook/display_rules.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/knative/test-infra/tools/webhook-apicoverage/coveragecalculator"
+)
+
+// display_rules.go contains all the display rules specified by knative serving to display json type like result display.
+
+// PackageDisplayRule rule specifies how package name needs to be displayed for json type like result display
+func PackageDisplayRule(packageName string) string {
+	if packageName != "" {
+		tokens := strings.Split(packageName, ".")
+		return strings.Join(tokens[len(tokens) - 2:], ".")
+	}
+	return packageName
+}
+
+// FieldDisplayRule rule specifies how the field needs to be displayed for json type like result display.
+func FieldDisplayRule(field *coveragecalculator.FieldCoverage) string {
+	var buffer strings.Builder
+	buffer.WriteString("\t" + field.Field)
```

Comment:
It seems counterintuitive that we're using string concatenation in the parameter to the string builder :-)

Reply:
I felt it would be more readable. Hence, have kept it that way, let me know if you strongly feel otherwise, will change it.